Libraries/LibWeb/WebAssembly/WebAssembly.cpp

```diff
@@ -157,6 +160,12 @@ namespace Detail {
 
 JS::ThrowCompletionOr<NonnullOwnPtr<Wasm::ModuleInstance>> instantiate_module(JS::VM& vm, Wasm::Module const& module, GC::Ptr<JS::Object> import_object)
 {
+    // 1. If module.imports is not empty, and importObject is undefined, throw a TypeError exception.
+    if (!module.import_section().imports().is_empty() && !import_object) {
+        return vm.throw_completion<JS::TypeError>(
+            "WebAssembly.instantiate: Import object must be defined when module has imports"sv);
```

Comment:
the prefix seems to be unnecessary?

Reply:
```
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/VM.h:208:30: error: no matching member function for call to 'create'
  208 |         auto completion = T::create(realm, forward<Args>(args)...);
      |                           ~~~^~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibWeb/WebAssembly/WebAssembly.cpp:165:19: note: in instantiation of function template specialization 'JS::VM::throw_completion<JS::TypeError, const char (&)[79]>' requested here
  165 |         return vm.throw_completion<JS::TypeError>(
      |                   ^
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:74:1: note: candidate function not viable: no known conversion from 'const char[79]' to 'String' for 2nd argument
   74 | JS_ENUMERATE_NATIVE_ERRORS
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Forward.h:61:5: note: expanded from macro 'JS_ENUMERATE_NATIVE_ERRORS'
   61 |     __JS_ENUMERATE(TypeError, type_error, TypeErrorPrototype, TypeErrorConstructor, void)                     \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:73:5: note: expanded from macro '__JS_ENUMERATE'
   73 |     DECLARE_NATIVE_ERROR(ClassName, snake_name, PrototypeName, ConstructorName)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:65:35: note: expanded from macro 'DECLARE_NATIVE_ERROR'
   65 |         static GC::Ref<ClassName> create(Realm&, String message);                   \
      |                                   ^              ~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:74:1: note: candidate function not viable: no known conversion from 'const char[79]' to 'StringView' for 2nd argument
   74 | JS_ENUMERATE_NATIVE_ERRORS
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Forward.h:61:5: note: expanded from macro 'JS_ENUMERATE_NATIVE_ERRORS'
   61 |     __JS_ENUMERATE(TypeError, type_error, TypeErrorPrototype, TypeErrorConstructor, void)                     \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:73:5: note: expanded from macro '__JS_ENUMERATE'
   73 |     DECLARE_NATIVE_ERROR(ClassName, snake_name, PrototypeName, ConstructorName)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:66:35: note: expanded from macro 'DECLARE_NATIVE_ERROR'
   66 |         static GC::Ref<ClassName> create(Realm&, StringView message);               \
      |                                   ^              ~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:74:1: note: candidate function not viable: requires 1 argument, but 2 were provided
   74 | JS_ENUMERATE_NATIVE_ERRORS
      | ^~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Forward.h:61:5: note: expanded from macro 'JS_ENUMERATE_NATIVE_ERRORS'
   61 |     __JS_ENUMERATE(TypeError, type_error, TypeErrorPrototype, TypeErrorConstructor, void)                     \
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:73:5: note: expanded from macro '__JS_ENUMERATE'
   73 |     DECLARE_NATIVE_ERROR(ClassName, snake_name, PrototypeName, ConstructorName)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/pavel/Develop/ladybird/Libraries/LibJS/Runtime/Error.h:64:35: note: expanded from macro 'DECLARE_NATIVE_ERROR'
   64 |         static GC::Ref<ClassName> create(Realm&);                                   \
      |                                   ^      ~~~~~~
1 error generated.
```

Reply:
I didn't manage to get things compiling without sv suffix

Reply:
@konradekk omg I'm so inattentive

Reply:
all good! as far as i can tell you’re doing great job—here and in other places! 🫡